src/event-handler.js

```diff
@@ -16,7 +16,10 @@ A non-ecommerce event has the following schema:
 function EventHandler(common) {
     this.common = common || {};
 }
-EventHandler.prototype.logEvent = function(event) {};
+EventHandler.prototype.logEvent = function(event) {
+    if (event.EventName == "click" || event.EventName == "change" || event.EventName == "submit") return;
+    window.heap.track(event.EventName, event.EventAttributes);
+};
```

Comment:
You should be using `===` to compare strings instead of `==`.

```suggestion
    if (event.EventName === "click" || event.EventName === "change" || event.EventName === "submit") return;
    window.heap.track(event.EventName, event.EventAttributes);
};
```